### ISSUE TITLE
Add Flow coverage for tracking code

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#6968]
We should add coverage to the tracking code, a.k.a.  `src/core/tracking.js`.

**For QA:** please make sure GA events are still sent (for example, on an add-on detail page) and also see if Hybrid Content Telemetry works in Disco Pane (-dev). That should be enough.

**Comment by ioanarusiczki** [comment]
@willdurand Is the Hybrid Content Telemetry from here https://github.com/mozilla/addons-frontend/issues/6867#issuecomment-438633992 ?

**Comment by willdurand** [comment]
@ioanarusiczki yes, exactly

**Comment by ioanarusiczki** [comment]
@willdurand 
- I verified if Hybrid Content Telemetry works with FF65(Win10) with instructions from https://github.com/mozilla/addons-frontend/issues/6867#issuecomment-438633992  
Disco Pane -dev Telemetry and it looks alright. I did not receive errors.
![telemetry with ff65](https://user-images.githubusercontent.com/33448286/48830317-f9f34f00-ed7c-11e8-9915-464f5529777d.png)

- I verified if GA are sent from the add-on detail page with FF63 ( release and unbranded versions) (Win10).  The Console (the Network tab) indicates GA requests when an add-on is installed, removed, links from the More information section are clicked.  Also every time the user lands on a detail page GA requests are present.

I did not see GA requests when an add-on is reported for abuse or the user makes a rating/review 
( except when deleting a review ).  If that is alright then I can mark issue verified.
